src/ui-config/permitConfig.ts

```diff
@@ -46,4 +54,9 @@ export const permitByChainAndToken: {
     '0x2416092f143378750bb29b79ed961ab195cceea5': true, // ezETH
     '0xb5bedd42000b71fdde22d3ee8a79bd49a568fc8f': true, // wstETH
   },
+  [ChainId.sonic]: {
+    '0x50c42dEAcD8Fc9773493ED674b675bE577f2634b': false, // WETH
+    '0x039e2fB66102314Ce7b64Ce5Ce3E5183bc94aD38': false, // wS
+    '0x29219dd400f2Bf60E5a23d13Be72B486D4038894': false, // USDC.e
```

Comment:
Addresses need to be normalized to lower case, probably would be a good thing to add to the above comment